index.js

```diff
@@ -0,0 +1,15 @@
+import * as booksinput from './modules/booksinput.js';
+
+const btnAdd = document.getElementById('btnAdd');
```

Comment:
- [ ] Create a Luxon module and import it into the index page to show the current day, it is a project requirement 👇🏽 

> Display the current date below the navigation bar.

> For this requirement you should use a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)
> Download the library into your project using npm.
> Import it as a module.
> See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). Your task here is to use a new tool effectively with little externally curated support. Exactly as you will do in your future job.